lib/cisco_node_utils/cmd_ref/bgp_neighbor.yaml

```diff
@@ -8,7 +8,7 @@ _template:
   context:
     - "router bgp <asnum>"
     - "(?)vrf <vrf>"
-    - "neighbor <nbr>"
+    - "(?)neighbor <nbr>"
```

Comment:
Both were marked as optional before your yaml refactoring, and the minitest failed with it marked mandatory.  I can double check that, though.